### ISSUE TITLE
VERB / ADJ

### ISSUE THREAD

**Issue by bguil** [surfacesyntacticud/SUD_French-GSD#2]
In **SUD_French-GSD**, construction like *est situé* is wrongly annotated in many cases ([Grew-match](http://match.grew.fr/?corpus=SUD_French-GSD@2.6&custom=5f685cf4e3479&clustering=e.label)).

In most of the cases, the relation is `comp:aux@tense` is used. This  is wrong because the whole sentence is in a present tense. For instance:

> *L'ambassade est située à Paris.*

![Screenshot 2020-09-21 at 10 21 16](https://user-images.githubusercontent.com/687155/93745668-32237d00-fbf4-11ea-838e-da6b2d947b3b.png)

Two options:

* **A**: *est* -[`comp:aux@pass`]-> *située*:`VERB`
* **B**: *est* -[`comp:pred`]-> *située*:`ADJ`

With @perrier54, we prefer solution **B** because it is not possible to have and active reformulation.

**Remarks**:
 * *situer* is used in a similar way as a modifier ([Grew-match](http://match.grew.fr/?corpus=SUD_French-GSD@latest&custom=5f68605038db6)) and should also be annotated as `ADJ` in these cases.

**Comment by sylvainkahane** [comment]
> _Où se situe l'ambassade ?_
> _J'ai du mal à situer l'ambassade ?_

J'ai quand même l'impression qu'on est pas loin du verbe _situer_ dans votre exemple.
Mais c'est vrai qu'il y a aussi des cas où ça semble encore plus un adjectif :

> _L'ambassade est vraiment mal située._

Où est la frontière ?
Quels critères proposez-vous pour trancher ? Absence de passif ?

C'est vrai que:
> _Le gouvernement a décidé de situer son ambassade à Paris._

est un peu bizarre. 
Par contre,
> _L'ambassade, je la situe pas bien. C'est où ?_

me parait normal, mais ce n'est pas exactement le même sens.

OK pour ADJ mais on voit que ça sera compliqué.

**Comment by kimgerdes** [comment]
Je suis d'accord que c'est un cas limite. j'ai une petite préférence pour la solution B (verbe). 
d'une part,  `situé` n'est pas un adjective typique (pas de modif par `très`, pas de qualité, ...) d'autre part, la localisation `à Paris` me semble être un argument locatif assez standard, rare sur un adjectif. et finalement, on a qm pas mal de phrases à l'active du genre `Il a décidé de situer l'intrigue de son dernier polar dans la ville préfecture des Côtes-d'Armor`
https://www.google.com/search?&q=%22a+d%C3%A9cid%C3%A9+de+situer+%22
